cijoe/tests/tools/test_xnvme.py

```diff
@@ -75,14 +75,20 @@ def test_format(cijoe, device, be_opts, cli_args):
     assert not err
 
 
-@xnvme_parametrize(labels=["nvm"], opts=["be", "admin"])
+@xnvme_parametrize(labels=["nvm", "sanitize"], opts=["be", "admin"])
 def test_sanitize(cijoe, device, be_opts, cli_args):
     if be_opts["be"] == "linux" and be_opts["admin"] in ["block"]:
         pytest.skip(reason="[admin=block] does not implement sanitize")
+    if "ramdisk" in device["labels"]:
+        pytest.skip(reason="[be=ramdisk] does not implement sanitize")
```

Comment:
Maybe stupid question, but why is the reason "[be=ramdisk] (...)", when this is not the condition we are looking for? Should the if-statement be `if be_opts["be"] == "ramdisk"`, or should the reason maybe be changed? 
(and also, is ramdisk both as a label *and* a backend?)

Reply:
You are right, this has been fixed now